src/components/DocumentLibrary/DocumentLibrary.tsx

```diff
@@ -553,9 +552,7 @@ export const DocumentLibrary = (props: DocumentLibraryProps) => {
             (a.content_type || '').localeCompare(b.content_type || '')
           ),
         REVISION: (array) =>
-          array.sort((a, b) =>
-            a.item.revision_count.localeCompare(b.item.revisions?.length)
-          ),
+          array.sort((a, b) => a.revision_count - b.revision_count),
         LATEST: (array) =>
           array.sort((a, b) => a.is_private.localeCompare(b.is_private)),
```

Comment:
Given that `is_private` is a boolean, wouldn't this line still fail?

Reply:
Oh.  That is no longer used. I went ahead and removed it.